### ISSUE TITLE
The _mouse_exit function doesn't get call in the TextureButton if the LMB is pressed

### ISSUE THREAD

**Issue by radubolovan** [godotengine/godot#7430]
**Operating system or device - Godot version:** Windows 10 - 2.1.1 stable

**Issue description** (what happened, and what was expected):
- I made a simple scene with a Node2D as "root" and a "TextureButton" as a child.
- I added two textures for the "Pressed" and "Normal" (just black and white textures for the test) button's states
- I also added a script attached to the "root" object having the following code:
```
extends Node2D

func _on_TextureButton_mouse_enter():
	print("_on_TextureButton_mouse_enter")

func _on_TextureButton_mouse_exit():
	print("_on_TextureButton_mouse_exit")
```

**Steps to reproduce:**
1) Move the mouse cursor in the button's bounding rect (TextureButton class - I didn't test with other classes)
2) Press the LMB (**L**eft **M**ouse **B**utton)
3) Move the mouse cursor outside the button's bounding rect
4) Experience the issue: the "_mouse_exit" function doesn't get called

OBS: If you release the LMB, the function is called.

**Link to minimal example project** (optional but very welcome):
https://github.com/radubolovan/godot_mouse_test

**Comment by bojidar-bg** [comment]
Probably related/dupe of #6362 (in retrospect).

**Comment by akien-mga** [comment]
Tested in 3.0.6 and 3.1-alpha, it's no longer reproducible, so assumed fixed in 3.0.
